vcr/issuer/openid4vci/types.go

```diff
@@ -0,0 +1,36 @@
+package openid4vci
+
+import (
+	"github.com/nuts-foundation/go-did/vc"
+	"time"
+)
+
+// Flow is an active OpenID4VCI credential issuance flow.
```

Comment:
I designed it as such it can be extended with non-credential-issuer-initiated flows (so no offers), so "flow" was what I settled on (also considered "state" but that's already a non-matching OpenID concept)